### DISCUSSION TITLE
[SOLVED] After updating to NC 27, all office documents incl new are read-only

### DISCUSSION THREAD

**Discussion by Stooovie** [nextcloud/all-in-one#2799] [Questions] [answered]
Can someone guide me through troubleshooting this? All worked fine in NC 26. Now even the new documents that I create (I am an admin) are all read-only. Thanks. 

Edit: ah, I see this in Security and setup warnings. That's new. 

```
The database is missing some indexes. Due to the fact that adding indexes on big tables could take some time they were not added automatically. By running "occ db:add-missing-indices" those missing indexes could be added manually while the instance keeps running. Once the indexes are added queries to those tables are usually much faster.
Missing index "mounts_user_root_path_index" in table "oc_mounts".

```

After running 
sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ db:add-missing-indices 

the problem seemed to go away after restarting NC and I was able to edit and create new editable documents but after a few minutes, the  reappeared. 

SOLUTION: it's not a bug, it's a silly UX issue: I was opening the documents on an iPhone and the mobile office app really DOES display a "read-only" banner EVEN WHEN THE FILE IS EDITABLE. Why, I have no idea.

**Comment by szaimen** [comment, marked as answer]
> SOLUTION: it's not a bug, it's a silly UX issue: I was opening the documents on an iPhone and the mobile office app really DOES display a "read-only" banner EVEN WHEN THE FILE IS EDITABLE. Why, I have no idea.

Probably it would make sense to open an issue about this here: https://github.com/nextcloud/richdocuments/issues and please also add screenshots there!

**Comment by Stooovie** [comment]
Agreed, done, [here](https://github.com/nextcloud/richdocuments/issues/3025)
